src/qwen3_tts_test/voice_clone_batch.py

```
@@ -0,0 +1,133 @@
+        np.concatenate(out_wav, axis=0) if out_wav else np.zeros((0,), dtype=np.float32)
+    )
+
+    sf.write(str(out_path), merged, out_sr)
+    logger.info("保存完了: %s sr=%d lines=%d", out_path, out_sr, len(lines))
+
+
+if __name__ == "__main__":
+    try:
+        main()
+    except VoiceCloneError as err:
+        logging.getLogger("qwen_tts").error("VoiceCloneError: %s", err)
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)